lib/spack/spack/cmd/buildcache.py

```diff
@@ -614,9 +669,31 @@ def manifest_copy(manifest_file_list):
                 deduped_manifest[spec_hash] = copy_list
 
     for spec_hash, copy_list in deduped_manifest.items():
+        meta_src, meta_dst, archive_src, archive_dst = (None, None, None, None)
+        # figure out which src/dest pair is the metadata and which is the archive
         for copy_file in copy_list:
```

Comment:
This loop doesn't look quite right to me, but I could be missing something

Reply:
Do you want to be more specific? :stuck_out_tongue_winking_eye:

Reply:
I was missing something, the manifest file is always a list of two src/dest maps